### DISCUSSION TITLE
Hey guys - when you're going to release? :)

### DISCUSSION THREAD

**Discussion by upgreydd** [Budibase/budibase#5653] [General]
Hello. I can't wait for some already merged fixes - is there an option to change release strategy and apply not only monthly releases but some minor version weekly releases? Or would be great to have docker images for `develop` branch - cherry picking and rebuilding takes lot of time to maintain some fixes.

**Comment by mike12345567** [comment]
Hi @Upgreydd - usually we release hot fixes as and when we have them, at the minute `master` hasn't been released in a while as we have some pretty hefty fixes that we are still testing and have discovered some issues with - however it will be released next week as part of the monthly release.

Features will continue to be released on a monthly cycle as we've found that this works quite well for us.

As for images, we do regularly release the `proxy`, `apps` and `worker` images - you can find them here:
1. apps - https://hub.docker.com/r/budibase/apps/tags
2. worker - https://hub.docker.com/r/budibase/worker/tags
3. worker - https://hub.docker.com/r/budibase/proxy/tags

We recommend you backup your volumes/apps before using any of the `develop` images as these are still in QA and being tested, we cannot guarantee that there won't be some possible breaking changes within them that we haven't discovered yet. You can also find versions which have been released from master with hot fixes (for example at the minute `v1.0.132`) which are still in testing and will soon be released.

In your `docker-compose.yaml` you can update it to something like `image: budibase/apps:develop` to use a specific image.

**Reply by upgreydd** [reply]
thanks @mike12345567 I didn't know you're publishing to docker hub too - I've noticed only `budibase.docker.scarf.sh` server in `docker-compose.yaml` so probably that's why I didn't check tags. Thanks for the info, I'll try maintain it myself for sure :+1:
